Modules/System/Convert/src/Convert.Codeunit.al

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Provides helper functions to convert data
+/// </summary>
+codeunit 1485 "Convert"
+{
+    Access = Public;
+
+    var
+        ConvertImpl: Codeunit "Convert Impl.";
+
+    /// <summary>
+    /// Converts a text from one encoding to another.
+    /// </summary>
+    /// <param name="SrcCodepage">Code page identifier of the source.</param>
+    /// <param name="DstCodepage">Code page identifier of the output.</param>
+    /// <param name="Text">The text containing the characters to encode.</param>
+    procedure ConvertTextEncoding(SrcCodepage: Integer; DstCodepage: Integer; Text: Text): Text
```

Comment:
Maybe an overload with `TextEncoding`?

I can now see that AL's `TextEncoding` isn't quite the same as .Net's `Encoding`. So maybe new enum?

Reply:
It is more of a suggestion than a change request. It's a minor thing so don't bother.